linode/helper/http.go

```diff
@@ -0,0 +1,33 @@
+package helper
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+)
+
+// AddRootCAToTransport applies the cert file at the given path to the given *http.Transport
+func AddRootCAToTransport(cert string, transport *http.Transport) error {
```

Comment:
When I set the path starting with `~` to represent home directory, it can't recognize it as a valid path. Should we consider expanding the home directory for user?

Reply:
Good call! I'll address this shortly